### PR TITLE
Added documentation for installation on Archlinux, as well as Archlinux's logo

### PR DESCRIPTION
**Pull request by babolivier**
The Archlinux package now has a documentation, soon in a newspaper store near you.

### CODE REVIEW

File: src/documents/en/host/install/install-on-archlinux.html.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+<div style="height: 0; overflow: shown; text-align: right">
+<img alt="Logo d’Ubuntu" src="/assets/images/archlinux-logo.png">
+</div>
+
+# Install Cozy on Archlinux
+
+Cozy packages are available on the **AUR** (*Archlinux User Repository*) repository, which is not natively supported by Archlinux. In order to install Cozy, you will need an AUR helper, preferably among the ones listed [here](https://wiki.archlinux.org/index.php/AUR_helpers).
+
+In this documentation, we'll be using **Yaourt**, which you can install following [these instructions](https://archlinux.fr/yaourt) (in French).
```
Comment (nono) [inline src/documents/en/host/install/install-on-archlinux.html.md]: @babolivier what do you think of https://archlinux.fr/yaourt-en?

File: src/documents/en/host/install/install-on-archlinux.html.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Installing
+
+Once Yaourt is installed, and the package is chosen (we'll call it `cozy-xxx` from here for simplicity's sake), please enter the following command:
+
+```
+yaourt -S cozy-xxx
+```
+
+A moins que vous ne souhaitiez bidouiller le paquet, répondez par la négative quand on vous demande si vous voulez modifier les fichiers `PKGBUILD` et `cozy-xxx.install`, et par l'affirmative lorsqu'on vous demande si vous souhaitez lancer la compilation du paquet.
```
Comment (nono) [inline src/documents/en/host/install/install-on-archlinux.html.md]: @babolivier can you remove this sentence? It is translated below.

File: src/documents/en/host/install/install-on-archlinux.html.md (hunk 1 above)
Comment (babolivier) [inline src/documents/en/host/install/install-on-archlinux.html.md]: Thanks @nono, I didn't know this link.

File: src/documents/en/host/install/install-on-archlinux.html.md (hunk 2 above)
Comment (babolivier) [inline src/documents/en/host/install/install-on-archlinux.html.md]: Whoops, forgot to remove this one.

**Comment by babolivier** [comment]
Thanks @nono for the review and the feedback. Everything should be fine now.

**Pull request merged**